scripts/hunspell/isocpp.dic

```diff
@@ -244,6 +244,7 @@ incomplet
 incorrekt
 increment1
 Incrementable
+incscope
```

Comment:
What is in scope?

Reply:
the label in the markup, and I now see it is needs updating given the change in guidance.